### PR TITLE
Add instance class to control the lifetime of all contexts

### PR DESCRIPTION
**Pull request by fujunwei**

### CODE REVIEW

**Comment by huningxin** [comment]
There is a clang format error by CI. Please double check.

**Comment by fujunwei** [comment]
Thanks, done.

File: webnn.json [hunk 1]
```diff
@@ -15,6 +15,32 @@
     "See the License for the specific language governing permissions and",
     "limitations under the License."
   ],
+  "instance descriptor": {
+    "category": "structure",
+    "members": []
+  },
+  "instance": {
+    "category": "object",
+    "methods": [
```
Comment (huningxin) [inline webnn.json:24]: just one question, you mentioned other classes like NamedOperands will be managed by instance, how? And will they be created by instance?

Comment (huningxin) [reply webnn.json:24]: checked with @fujunwei , other classes will be managed by instance in following PRs.

**Review by huningxin — APPROVED** [review]
LGTM, thanks @fujunwei

**Pull request merged**